lib/kubernetes-deploy/kubernetes_resource/bugsnag.rb

```diff
@@ -2,10 +2,13 @@
 module KubernetesDeploy
   class Bugsnag < KubernetesResource
     TIMEOUT = 1.minute
+    PREDEPLOY = true
+    GROUP = 'stable.shopify.io'
+    VERSION = 'v1'
```

Comment:
We provide these values in code for statically defined classes. They can also be provided dynamically by annotations to CRDs.

Reply:
We should delete this class, actually... we 🔥 this CR a while ago.